### PR TITLE
summit speakers grid submitters support

### PR DESCRIPTION
**Pull request by romanetar**
ref https://tipit.avaza.com/project/view#!tab=task-pane&groupby=MyTaskDueDate&view=vertical&task=3210795&fileview=grid

Signed-off-by: romanetar [roman_ag@hotmail.com]

### CODE REVIEW

File: src/reducers/summit_submitters/summit-submitters-list-reducer.js [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+            return DEFAULT_STATE;
+        }
+        case REQUEST_SUBMITTERS_BY_SUMMIT: {
+            let { order, orderDir, term, page, perPage, ...rest } = payload;
+            return { ...state, order, orderDir, term, currentPage: page, perPage, ...rest }
+        }
+        case RECEIVE_SUBMITTERS_BY_SUMMIT: {
+            let { current_page, total, last_page } = payload.response;
+
+            let items = payload.response.data.map(s => {
```
Comment (smarcet) [inline src/reducers/summit_submitters/summit-submitters-list-reducer.js:67]: lets refactor this for both ( speakers and submitter) and use a single util method to render this

File: src/pages/summit_speakers/summit-speakers-list-page.js [hunk 2]
```diff
@@ -252,14 +316,27 @@ class SummitSpeakersListPage extends React.Component {
             { label: 'Alternate/Rejected', value: 'alternate_rejected' },
         ];
 
-        let emailFlowDDL = [
+        const speakerSubmitterSourceSelectorDDL = [
+            { label: 'Speakers', value: sources.speakers },
```
Comment (smarcet) [inline src/pages/summit_speakers/summit-speakers-list-page.js:320]: add i18n resources

**Review by smarcet — CHANGES_REQUESTED** [review]
@romanetar please review comments

File: src/pages/summit_speakers/summit-speakers-list-page.js (hunk 2 above)
Comment (romanetar) [reply src/pages/summit_speakers/summit-speakers-list-page.js:320]: done

File: src/reducers/summit_submitters/summit-submitters-list-reducer.js (hunk 1 above)
Comment (romanetar) [reply src/reducers/summit_submitters/summit-submitters-list-reducer.js]: done

**Comment by romanetar** [comment]
@smarcet review comments resolved

**Pull request merged**